Fix #1908:Changed comments in Proto files

Victor-Titan opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
- Fixes #1908 
- Changed org.oppia.android.android/ to org/oppia/android/android/ in the comments of the proto files

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

PTAL @prayutsu

Victor-Titan committed:
Fixed the incorrect paths written in the proto files

Sarthak2601 commented:
Hey @Victor-Titan! Thanks for contributing to Oppia :)
There are a couple of things that you can do to improve your pull requests - 
- Follow this [guide](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR).
- Always claim the issue first (by commenting on it or by self assigning) and then open a PR corresponding to that issue. Right now, the issue that you've opened up your PR for, has been assigned to someone else. 
- The title of you PR should be - `Fix #1908: Changed comments in Proto file`. 

Happy contributing 👍

anandwana001 commented on .idea/codeStyles/Project.xml:
```diff
@@ -5,6 +5,22 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```
We should not change this file. 
This change happens sometimes when our Android Studio coding style is not as same as the project. You can undo these changes and check `Editor->Code Style->scheme->Project`. Also, you can set code style for Java, Kotlin, XML to project. 

This way Android Studio will pick the code style from the project files and do not enforce the Android Studio code style on the project.

Victor-Titan replied:
Ok I'll do that

anandwana001 replied:
> @anandwana001 I made the changes, can you please check and merge if it's ok?

The codestyle thing has not been updated yet. Please check the file and I will review again.

Victor-Titan replied:
I've updated the codestyle as you mentioned. 
![image](https://user-images.githubusercontent.com/54766268/94466307-e027bc00-01de-11eb-8cf6-f34920196ad4.png)

anandwana001 replied:
looks good, Thanks

anandwana001 commented on build.gradle:
```diff
@@ -20,6 +20,7 @@ buildscript {
 }
 
 allprojects {
+
```
This space is not needed. You can remove this empty line.

anandwana001 reviewed (changes requested):
Thanks @Victor-Titan 
Added a few points to check on. Let me know if you need any help on these.

Victor-Titan committed:
Removed extra code in Project.xml file and an extra line in build.gradle

Victor-Titan commented:
@anandwana001 I made the changes, can you please check and merge if it's ok?

Victor-Titan committed:
Removed extra code in Project.xml file

Sarthak2601 commented on .idea/codeStyles/Project.xml:
```diff
@@ -161,4 +161,4 @@
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Please don't push `.idea` file changes. You can simply delete this file from your PR.

Victor-Titan replied:
Done, I didn't know we could delete that, else I would've just done that instead of changing the code

Sarthak2601 replied:
Wait, this is incorrect actually. What you've done might lead to deleting of the file from our codebase itself. I'll defer to @anandwana001 on this.

anandwana001 replied:
I think the steps you should take here is:
1. Update your branch with the latest develop branch. (pull + push or fetch + merge + commit + push)
2. Now, the `.idea/codeStyles/Project.xml` will be added to your branch, as coming from the latest develop branch.
3. Now, you will see the file is same as this file https://github.com/oppia/oppia-android/blob/develop/.idea/codeStyles/Project.xml and no changes will be shown into your branch, meaning Android Studio is not updating this file. 
4. Now, in this PR you will only see 3 files changed which is our proto file where you had solved this issue.

As per @Sarthak2601 `deleting this file from your pr` means you can rollback the changes in this file and then commit.

Sarthak2601 reviewed (changes requested):
Thanks @Victor-Titan. Suggested just one change.

Victor-Titan commented:
PTAL @Sarthak2601 @anandwana001 I've implemented the changes you asked

anandwana001 reviewed (changes requested):
We don't need to delete `.idea/codeStyles/Project.xml` but keep it unchanged. 
Merge your branch with the latest `develop` branch.

Victor-Titan committed:
Merge remote-tracking branch 'upstream/develop' into fix-comments-broken

Victor-Titan committed:
Merged new changes from develop branch to local repo

Victor-Titan commented:
I pulled the code from the main repo and commited again, but the tests showed a build fail in the part of the code I pulled. If there was a build fail issue then how was that code merged to develop? PTAL @anandwana001

anandwana001 commented:
> I pulled the code from the main repo and commited again, but the tests showed a build fail in the part of the code I pulled. If there was a build fail issue then how was that code merged to develop? PTAL @anandwana001

This particular test is passing for me. And also passing on the latest develop branch. Let's just first clear the `.idea/codeStyles/Project.xml` issue we are having in this PR and then we can see if the test fails or not, It's not dependent on that particularly but then we can update the branch again. Also, force push is not much recommended.

aggarwalpulkit596 commented:
@Victor-Titan hey victor let's do one thing let's clean up your fork first. For that you can do things either delete the existing one and make new one or add a remote

prayutsu commented:
@Victor-Titan Have you installed the Kotlin-plugin in your Android Studio, if you want to fix that Project.xml issue, you can uninstall it, it's not of much use. This bug is actually shipped with the latest version of the Kotlin Plug-in.

prayutsu commented:
> I pulled the code from the main repo and commited again, but the tests showed a build fail in the part of the code I pulled. If there was a build fail issue then how was that code merged to develop? PTAL @anandwana001

This problem has been experienced by a lot of developers recently, and as for now the best solution to avoid this is to uninstall the Kotlin Plug-in, if you haven't installed Kotlin Plug-in then let me know, I'll suggest another way to fix it.

Victor-Titan commented:
@prayutsu @aggarwalpulkit596 @anandwana001 I removed the Kotlin plugin and also deleted my existing fork and made a new one, but when I went ahead with committing, the .idea/codeStyles/Project.xml file still got changed. Please help. While I can remove that file while committing code, doing so everytime seems tedient

aggarwalpulkit596 commented:
This PR should be closed in that case ? And it won't be pushed for sure for a new fork I'm pretty sure about that this entire folder has been added to gitignore so that case shouldn't be there

prayutsu commented:
> @prayutsu @aggarwalpulkit596 @anandwana001 I removed the Kotlin plugin and also deleted my existing fork and made a new one, but when I went ahead with committing, the .idea/codeStyles/Project.xml file still got changed. Please help. While I can remove that file while committing code, doing so everytime seems tedient

Okay

I want you to confirm one thing, if you have successfully uninstalled Kotlin plug-in, then please try deleting that extra piece of code one last time and try committing changes.

If it still appears again then please follow the steps - 

You have to restore your Android Studio Settings to default. Assuming that you are on windows, I 
am sharing a link to do this.

https://stackoverflow.com/questions/20582577/how-to-reset-to-default-configuration-in-android-studio/31228003

If you are on linux or mac then do let me know.

Also after doing this don't forget to delete the extra code that is getting appeared again and again. After deleting one last time, you should not encounter this error.

prayutsu commented:
Also, if you can please share the Screenshot to confirm that you have successfully uninstalled the Kotlin plug-in, because I am quite sure that this problem comes with the latest release of Kotlin plug-in.

prayutsu commented:
@Victor-Titan Also, Android Studio will prompt you again and again by showing the dialog bubble in the bottom right corner but do not install the Kotlin Plugin for now.

Victor-Titan commented:
@prayutsu I can't seem to uninstall the Kotlin Plugin, I'm only allowed to disable it. When I tried disabling it, a message popped up showing that it was part of a bundle of other google plugins including Firebase(I've not installed any of them, I think they were there to start with) and when I disabled them all and restarted, I got a message like this
![image](https://user-images.githubusercontent.com/54766268/94568174-cee7b980-0289-11eb-8005-b4158e249c7b.png)

I'm on Linux. Also, there was another Kotlin Plugin which I had installed back when I started Android dev in Kotlin. I think I uninstalled that one.

Victor-Titan commented:
@prayutsu @anandwana001 I'm closing this PR as I created a new PR when I pushed code to a new fork as @aggarwalpulkit596 suggested.
